config/pgpass-mainnet

```diff
@@ -1 +1 @@
-/var/run/postgresql:5432:cexplorer:*:*
+localhost:5432:cexplorer:*:*
```

Comment:
Possibly leftover local diff?

Reply:
good spot yes it was!! there will be another too, let me take them out :)